backend/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "backend",
+  "scripts": {
+    "start": "node lib/app.js",
+    "dev": "ts-node src/app.ts"
```

Comment:
consider using [nodemon](https://www.npmjs.com/package/nodemon) so you dont have to manually run `yarn dev` every time you make changes.
If you decide to add nodemon as a dependency, change this line to `nodemon --exec ts-node src/app.ts`

Reply:
Alright, I'll try it out.

Reply:
Nice!